domain/domain.go

```diff
@@ -674,9 +674,8 @@ func (do *Domain) Init(ddlLease time.Duration, sysFactory func(*Domain) (pools.R
 				DialOptions: []grpc.DialOption{
 					grpc.WithBackoffMaxDelay(time.Second * 3),
 					grpc.WithKeepaliveParams(keepalive.ClientParameters{
```

Comment:
It is better to use `DialKeepAliveTime` and `DialKeepAliveTimeout` directly in clientv3.Config instead of `grpc.WithKeepaliveParams`. Just like `kv.go`.